app/components/layout/top-nav.tsx

```diff
@@ -1,240 +1,104 @@
-import { Form } from '@remix-run/react';
+import React, { useState } from 'react';
 
-import { useCurrentUser } from '~/hooks/useCurrentUser';
+const TopNav: React.FC = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
 
-export default function TopNav() {
-  const currentUser = useCurrentUser();
```

Comment:
Can we hide the Sign-up / Login buttons if the user is already logged in and instead show a logout button? We can also pair-program this next session together! :) CC @tonydangblog

Reply:
As mentioned in my approval comment, I'm ok with either adding back this functionality in this PR or doing it in a follow-up PR :)